src/bidderSettings.js

```diff
@@ -16,9 +16,12 @@ export class ScopedSettings {
    * @param path {String}
    * @returns {*}
    */
-  get(scope, path) {
+  get(scope, path, parentScope) {
```

Comment:
Can you explain the new argument in the method's jsdoc?

Reply:
Consider below scenario
```
pbjs.aliasBidder('pubmatic', 'foobar');
pbjs.bidderSettings = {
     pubmatic: {
        allowUnknownBidderCodes: true,
        allowedUnknownBidderCodes: ['unexpected']
     }
}
```

Here when we search for bidderSettings of `foobar` we wont find it, since `bidderSettings` are applied to its adapter i.e. `pubmatic`. To make adapter settings accessible to `bidder` i have made this change.
As explained in my comment yesterday the priority of `bidderSettings.get` function will be

1. Bidder level
2. Adapter level
3. Standard

Same thing was highlighted in IMPACTS of [8129](https://github.com/prebid/Prebid.js/issues/8129)

Reply:
yes that makes sense, what I meant is - can you update the method's jsdoc comment? so that someone who didn't look at this pr knows what `parentScope` is :)